src/game/scenes/vs.c

```diff
@@ -617,7 +618,7 @@ int vs_create(scene *scene) {
     scene_set_render_cb(scene, vs_render);
     scene_set_render_overlay_cb(scene, vs_render_overlay);
     scene_set_input_poll_cb(scene, vs_input_tick);
-    scene_set_dynamic_tick_cb(scene, vs_dynamic_tick);
+    // scene_set_dynamic_tick_cb(scene, vs_dynamic_tick);
```

Comment:
No longer needed ?

Reply:
it never seemed necessary

Reply:
Ok. Just remove the line and the tick function completely ? We can find it in git history if it's needed.

Reply:
Fixed, I just forgot to remove it.